.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: release
+
+on: workflow_dispatch
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
```

Comment:
We should consider releasing on a given version. Following the latest Ubuntu can cause release issues, but on the other hand, this way we realize these problems ASAP.

Reply:
We also use 'ubuntu-latest' in the test, because bldr should work on the latest ubuntu. The releaser job will bumb the version and push this change to the repo, which will trigger a test. So I think to fix ubuntu version will not help.